### PR TITLE
Many fixes mentioned in #1390

### PR DESCRIPTION
**Pull request by zoffixznet**

### CODE REVIEW

File: perl6.html.markdown [hunk 1]
```diff
@@ -75,7 +75,7 @@ say @array; #=> a 6 b
 #  except they get "flattened" (hash context), removing duplicated keys.
 my %hash = 1 => 2,
            3 => 4;
-my %hash = autoquoted => "key", # keys get auto-quoted
+my %hash = foo => "bar", # keys get auto-quoted if they're simple enough
```
Comment (vendethiel) [inline perl6.html.markdown]: I'm worried this line is too wide for the website, but we'll see. Also: "simple enough"?

Comment (zoffixznet) [inline perl6.html.markdown]: Updated to remove "if they're simple enough." Originally, I wanted to clarify that not just any key would get autoquoted, but explaining that becomes too verbose. I think in the context of a short tutorial that can indeed be omited.

File: perl6.html.markdown [hunk 2]
```diff
@@ -608,27 +609,26 @@ sub foo {
...
 
 ### Object Model
 
-## Perl 6 has a quite comprehensive object model
 # You declare a class with the keyword `class`, fields with `has`,
-# methods with `method`. Every field to private, and is named `$!attr`,
-# but you have `$.` to get a public (immutable) accessor along with it.
-# (using `$.` is like using `$!` plus a `method` with the same name)
+# methods with `method`. Every attribute that is private is named `$!attr`.
+# Immutable public methods are named `$.attr`
```
Comment (vendethiel) [inline perl6.html.markdown]: methods?

File: perl6.html.markdown [hunk 3]
```diff
@@ -857,12 +854,12 @@ for ^5 -> $a {
 #  program. When the program is compiled, when a for loop runs, when you leave a
 #  block, when an exception gets thrown ... (`CATCH` is actually a phaser !)
 # Some of them can be used for their return values, some of them can't
-#  (those that can have a "[*]" in the beginning of their explanation text).
+#  (those that can, have a "[*]" in the beginning of their explanation text).
```
Comment (vendethiel) [inline perl6.html.markdown]: -1

File: perl6.html.markdown (hunk 2 above)
Comment (zoffixznet) [inline perl6.html.markdown]: Good catch. Should be `attributes`. Updated.

File: perl6.html.markdown (hunk 3 above)
Comment (zoffixznet) [inline perl6.html.markdown]: Removed.

File: perl6.html.markdown [hunk 4]
```diff
@@ -891,12 +891,12 @@ for ^5 {
...
 COMPOSE { "When a role is composed into a class. /!\ NOT YET IMPLEMENTED" }
 
-# They allow for cute trick or clever code ...:
-say "This code took " ~ (time - CHECK time) ~ "s to run";
+# They allow for cute tricks or clever code ...:
+say "This code took " ~ (time - CHECK time) ~ "s to compile";
 
 # ... or clever organization:
 sub do-db-stuff {
-  ENTER $db.start-transaction; # New transaction everytime we enter the sub
```
Comment (vendethiel) [inline perl6.html.markdown]: I guess we need a new ENTER example now, though?

File: perl6.html.markdown [hunk 5]
```diff
@@ -1242,14 +1241,14 @@ so 'foo!' ~~ / <-[ a..z ] + [ f o ]> + /; # True (the + doesn't replace the left
 # Group: you can group parts of your regexp with `[]`.
 # These groups are *not* captured (like PCRE's `(?:)`).
 so 'abc' ~~ / a [ b ] c /; # `True`. The grouping does pretty much nothing
-so 'fooABCABCbar' ~~ / foo [ A B C ] + bar /;
+so 'fooABCABCbar' ~~ / foo [ ABC ] + bar /;
```
Comment (vendethiel) [inline perl6.html.markdown]: there's no point in making it a group anymore

**Comment by vendethiel** [comment]
cool stuff :+1:

File: perl6.html.markdown (hunk 4 above)
Comment (zoffixznet) [inline perl6.html.markdown]: Do we? There are no examples for `BEGIN`, `INIT`, `END`. I think it's clear what `ENTER` does and IMO the phaser isn't that useful to warrant an example.

File: perl6.html.markdown (hunk 5 above)
Comment (zoffixznet) [inline perl6.html.markdown]: Removed this change

File: perl6.html.markdown (hunk 4 above)
Comment (vendethiel) [inline perl6.html.markdown]: no, it should be okay. thanks :-)

**Comment by vendethiel** [comment]
:heart: :heart_decoration:

**Pull request merged**